go/cmd/dolt/commands/dump.go

```diff
@@ -90,7 +90,7 @@ func (cmd DumpCmd) ArgParser() *argparser.ArgParser {
 	ap.SupportsString(filenameFlag, "fn", "file_name", "Define file name for dump file. Defaults to `doltdump.sql`.")
 	ap.SupportsString(directoryFlag, "d", "directory_name", "Define directory name to dump the files in. Defaults to `doltdump/`.")
 	ap.SupportsFlag(forceParam, "f", "If data already exists in the destination, the force flag will allow the target to be overwritten.")
-	ap.SupportsFlag(batchFlag, "", "Returns batch insert statements wherever possible.")
+	ap.SupportsFlag(batchFlag, "", "Turns off batching insert statements wherever possible.")
```

Comment:
If we're going to make this change, then change the flag to --no-batch and make the existing --batch a no-op